src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/actions/ModifyCacheMaxSizeAction.java

```diff
@@ -120,7 +120,8 @@ public String summary() {
 
   // Generates action from summary. Passing in appContext because it contains dynamic settings
   public static ModifyCacheMaxSizeAction fromSummary(String jsonRepr, AppContext appContext) {
-    final JsonObject jsonObject = JsonParser.parseString(jsonRepr).getAsJsonObject();
+    final JsonParser jsonParser = new JsonParser();
+    final JsonObject jsonObject = jsonParser.parse(jsonRepr).getAsJsonObject();
```

Comment:
Can we have a static JsonParser that is allocated once in a separate Util class and reused by both these Action classes?